test/e2e/kubernetes_config_generate_test.go

```diff
@@ -425,11 +425,10 @@ func TestExportNetworkContainerAndPeerings(t *testing.T) {
 	awsPeeringID := s.generator.generateAWSPeering(awsContainerID, awsAppVPC)
 	defer s.generator.deletePeering(awsPeeringID)
 
-	// TODO: debug why this works locally but fails in Evergreen even with the same Azure parameters
-	// azureAppVPC := s.generator.generateAzureVPC("10.64.0.0/21", "northeurope")
-	// defer s.generator.deleteAzureVPC(azureAppVPC)
-	// azurePeeringID := s.generator.generateAzurePeering(azureContainerID, azureAppVPC)
-	// defer s.generator.deletePeering(azurePeeringID)
+	azureAppVPC := s.generator.generateAzureVPC("10.64.0.0/21", "northeurope")
+	defer s.generator.deleteAzureVPC(azureAppVPC)
+	azurePeeringID := s.generator.generateAzurePeering(azureContainerID, azureAppVPC)
```

Comment:
non-blocking: can this potentially leave leftovers if it gets an error?

Reply:
how? the next line is a defer to clean up the peering.
Or do you mean something else?